package.json

```diff
@@ -33,14 +33,18 @@
   },
   "devDependencies": {
     "browser-sync": "^2.24.5",
+    "cherio": "^1.0.0-rc.2",
     "del": "^3.0.0",
-    "gh-pages": "^1.2.0",
     "gulp": "^4.0.0",
-    "gulp-rename": "^1.2.2",
+    "gh-pages": "^1.2.0",
+    "gulp-rename": "^1.3.0",
     "gulp-replace": "^1.0.0",
-    "gulp-zip": "^4.0.0",
+    "gulp-zip": "^4.1.0",
+    "gulp-util": "^3.0.8",
+    "highlight.js": "^9.12.0",
     "merge-stream": "^1.0.0",
-    "path-exists-cli": "^1.0.0"
+    "path-exists-cli": "^1.0.0",
+    "run-sequence": "^2.2.1"
```

Comment:
is this a new package?

I do not see its use in PR

Reply:
It’s not used anymore since moving to Gulp 4.